### ISSUE TITLE
Error while upgrading to ng9/ivy with ng update

### ISSUE THREAD

**Issue by LanderBeeuwsaert** [angular/angular#33718]
I ran it twice.
The second time I cleared out my node_modules folder before starting the run.

used:
npm install --no-save @angular/cli@^8.3.18
ng update @angular/cli @angular/core --next

seems to fail on: experimental mdc-menu somewhere?

If I try to run after that, it doesn't load the application.

Best regards

C:\Lander\WS\acro-companion>npm install --no-save @angular/cli@^8.3.18

> grpc@1.23.3 install C:\Lander\WS\acro-companion\node_modules\grpc
> node-pre-gyp install --fallback-to-build --library=static_library

node-pre-gyp WARN Using request for node-pre-gyp https download
[grpc] Success: "C:\Lander\WS\acro-companion\node_modules\grpc\src\node\extension_binary\node-v72-win32-x64-unknown\grpc_node.node" is installed via remote

> node-sass@4.13.0 install C:\Lander\WS\acro-companion\node_modules\node-sass
> node scripts/install.js

Cached binary found at C:\Users\Lander\AppData\Roaming\npm-cache\node-sass\4.13.0\win32-x64-72_binding.node

> core-js@3.2.1 postinstall C:\Lander\WS\acro-companion\node_modules\@angular-devkit\build-angular\node_modules\core-js
> node scripts/postinstall || echo "ignore"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

> core-js@3.2.1 postinstall C:\Lander\WS\acro-companion\node_modules\@firebase\polyfill\node_modules\core-js
> node scripts/postinstall || echo "ignore"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

> protobufjs@6.8.8 postinstall C:\Lander\WS\acro-companion\node_modules\protobufjs
> node scripts/postinstall

> angular-tree-component@8.5.1 postinstall C:\Lander\WS\acro-companion\node_modules\angular-tree-component
> opencollective-postinstall || true

Thank you for using angular-tree-component!
If you rely on this package, please consider supporting our open collective:
> https://opencollective.com/angular-tree-component/donate

> node-sass@4.13.0 postinstall C:\Lander\WS\acro-companion\node_modules\node-sass
> node scripts/build.js

Binary found at C:\Lander\WS\acro-companion\node_modules\node-sass\vendor\win32-x64-72\binding.node
Testing binary
Binary is fine

> @angular/cli@8.3.18 postinstall C:\Lander\WS\acro-companion\node_modules\@angular\cli
> node ./bin/postinstall/script.js

npm WARN The package x----x----x is included as both a dev and production dependency.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules\webpack-dev-server\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x
64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules\watchpack\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x
64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules\jest-haste-map\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x
64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules\@angular\compiler-cli\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x
64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.0.7 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.0.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x
64"})

+ @angular/cli@8.3.18
added 2500 packages from 1058 contributors and audited 886190 packages in 136.688s

2 packages are looking for funding.
Run "npm fund" to find out more.

C:\Lander\WS\acro-companion>ng update @angular/cli @angular/core --next
The installed Angular CLI version is older than the latest published version.
Installing a temporary version to perform the update.
Installing packages for tooling via npm.

> @angular/cli@9.0.0-rc.1 postinstall C:\Users\Lander\AppData\Local\Temp\.ng-temp-packages-MDo5l2\node_modules\@angular\cli
> node ./bin/postinstall/script.js

+ @angular/cli@9.0.0-rc.1
added 254 packages from 206 contributors in 22.561s
Installed packages for tooling via npm.
Using package manager: 'npm'
Collecting installed dependencies...
Found 88 dependencies.
Fetching dependency metadata from registry...
                  Package "@angular/core" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/compiler-cli" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/forms" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/compiler" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/animations" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/platform-browser-dynamic" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/service-worker" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/router" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/common" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/platform-browser" has a missing peer dependency of "tslib" @ "^1.10.0".
    Updating package.json with dependency @angular/cli @ "9.0.0-rc.1" (was "8.3.18")...
    Updating package.json with dependency @angular/core @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/language-service @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular-devkit/build-angular @ "0.900.0-rc.1" (was "0.803.18")...
    Updating package.json with dependency @angular/compiler-cli @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/forms @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/compiler @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/animations @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/platform-browser-dynamic @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/service-worker @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/router @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/common @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency typescript @ "3.6.4" (was "3.5.3")...
    Updating package.json with dependency @angular/platform-browser @ "9.0.0-rc.1" (was "8.2.13")...
UPDATE package.json (7026 bytes)
√ Packages installed successfully.
** Executing migrations of package '@angular/cli' **

> Update an Angular CLI project to version 9.
UPDATE angular.json (20187 bytes)
UPDATE tsconfig.json (576 bytes)
UPDATE src/tsconfig.app.json (261 bytes)
UPDATE package.json (7023 bytes)
√ Packages installed successfully.
√ Migration succeeded.

> Update lazy loading syntax to use dynamic imports.
√ Migration succeeded.

** Executing migrations of package '@angular/core' **

> Static flag migration.
  Removes the `static` flag from dynamic queries.
  As of Angular 9, the "static" flag defaults to false and is no longer required for your view and content queries.
  Read more about this here: https://v9.angular.io/guide/migration-dynamic-flag
UPDATE src/app/create/modals/components/one-individual.component.ts (26328 bytes)
UPDATE src/app/create/modals/nat-individuals-selector.component.ts (6859 bytes)
UPDATE src/app/generic-components/sheet-purchase/sheet-purchase.component.ts (22080 bytes)
UPDATE src/app/organisation-admin/coach-sheet-list/coach-sheet-list.component.ts (27586 bytes)
UPDATE src/app/generic-components/sheet-search/sheet-search.component.ts (31524 bytes)
UPDATE src/app/judging/dj-judge-component/dj-judge.component.ts (47249 bytes)
UPDATE src/app/organisation-admin/sheet-overview/sheet-overview.component.ts (10625 bytes)
UPDATE src/app/organisation-admin/organisation-admin/organisation-admin.component.ts (27972 bytes)
UPDATE src/app/create/sheet-view.component.ts (20498 bytes)
UPDATE src/app/create/modals/group-mount-selector.component.ts (36269 bytes)
UPDATE src/app/create/modals/save-sheet-modal.component.ts (8131 bytes)
UPDATE src/app/create/modals/create-new-sheet.component.ts (32331 bytes)
UPDATE src/app/judging/ej-judge-component/ej-judge.component.ts (34719 bytes)
UPDATE src/app/judging/cjp-judge-component/cjp-judge.component.ts (33649 bytes)
UPDATE src/app/create/modals/pair-mount-selector.component.ts (32951 bytes)
UPDATE src/app/create/modals/dynamic-selector.component.ts (42016 bytes)
UPDATE src/app/create/modals/group-static-selector.component.ts (61607 bytes)
UPDATE src/app/create/modals/dynamic-double-landing.component.ts (50012 bytes)
UPDATE src/app/app.component.ts (14647 bytes)
UPDATE src/app/create/print/print-sheet-view.component.ts (9569 bytes)
UPDATE src/app/load/load.component.ts (28157 bytes)
UPDATE src/app/personal-invoice-list/personal-invoice-list.component.ts (8200 bytes)
UPDATE src/app/login/sign-in/sign-in.ts (23099 bytes)
UPDATE src/app/login/register/register.ts (21131 bytes)
UPDATE src/app/login/forgot-password/forgot-password.ts (8301 bytes)
UPDATE src/app/developer-admin/print-preview/print-preview.component.ts (37490 bytes)
UPDATE src/app/application-admin/user-admin/user-admin.component.ts (18390 bytes)
UPDATE src/app/application-admin/error-admin/error-admin.component.ts (19591 bytes)
UPDATE src/app/application-admin/create-free-form-invoice/create-free-form-invoice.component.ts (13559 bytes)
UPDATE src/app/application-admin/invoice-admin/invoice-admin.component.ts (20966 bytes)
UPDATE src/app/judging/kiss-judge-component/kiss-judge.component.ts (9209 bytes)
UPDATE src/app/judging/aj-judge-component/aj-judge.component.ts (27247 bytes)
UPDATE src/app/organisation-admin/session-panel-list/session-panel-list.component.ts (11102 bytes)
UPDATE src/app/organisation-admin/competition-administration/competition-administration.component.ts (19933 bytes)
UPDATE src/app/organisation-admin/panel-form/panel-form.component.ts (13625 bytes)
UPDATE src/app/organisation-admin/federation-administration/federation-administration.component.ts (19159 bytes)
UPDATE src/app/competition-calendar/competition-calendar/competition-calendar.component.ts (26194 bytes)
UPDATE src/app/artistic/artistic-create-sheet/artistic-create-sheet.component.ts (7240 bytes)
UPDATE src/app/artistic/artistic-judging/artistic-judging.component.ts (7375 bytes)
UPDATE src/app/tumbling/tumbling-create-sheet/tumbling-create-sheet.component.ts (7505 bytes)
UPDATE src/app/tumbling/tumbling-judging/tumbling-judging.component.ts (7112 bytes)
UPDATE src/app/titlebar/titlebar.component.ts (28302 bytes)
√ Migration succeeded.

> Missing @Injectable migration.
  In Angular 9, enforcement of @Injectable decorators for DI is a bit stricter.
  Read more about this here: https://v9.angular.io/guide/migration-injectable
√ Migration succeeded.

> ModuleWithProviders migration.
  In Angular 9, the ModuleWithProviders type without a generic has been deprecated.
  This migration adds the generic where it is missing.
  Read more about this here: https://v9.angular.io/guide/migration-module-with-providers
√ Migration succeeded.

> NGCC postinstall migration.
  Adds an ngcc invocation to npm/yarn's postinstall script.
  Read more about this here: https://v9.angular.io/guide/migration-ngcc
UPDATE package.json (7129 bytes)
/ Installing packages...
> acro-companion@1.0.0 postinstall C:\Lander\WS\acro-companion
> ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points

Compiling @angular/cdk/keycodes : es2015 as esm2015
Compiling @angular/cdk/testing : es2015 as esm2015
Compiling @angular/service-worker/config : es2015 as esm2015
Compiling @angular/animations : es2015 as esm2015
Compiling @angular/compiler/testing : es2015 as esm2015
Compiling ng2-ace-editor : module as esm5
Compiling @angular/core : es2015 as esm2015
Compiling angular-tree-component : module as esm5
Compiling @angular/cdk/coercion : es2015 as esm2015
Compiling @angular/common : es2015 as esm2015
Compiling @angular/cdk : es2015 as esm2015
Compiling @angular/animations/browser : es2015 as esm2015
Compiling @angular/cdk/collections : es2015 as esm2015
Compiling @angular/core/testing : es2015 as esm2015
Compiling @angular/cdk/portal : es2015 as esm2015
Compiling @angular/animations/browser/testing : es2015 as esm2015
Compiling ngx-page-scroll-core : es2015 as esm2015
Compiling @angular/cdk/observers : es2015 as esm2015
Compiling @angular/cdk/platform : es2015 as esm2015
Compiling @angular/cdk-experimental : es2015 as esm2015
Compiling @angular/cdk/accordion : es2015 as esm2015
Compiling @angular/cdk/bidi : es2015 as esm2015
Compiling @angular/platform-browser : es2015 as esm2015
Compiling @angular/fire : es2015 as esm2015
Compiling @angular/cdk/a11y : es2015 as esm2015
Compiling @angular/forms : es2015 as esm2015
Compiling @angular/common/http : es2015 as esm2015
Compiling @angular/cdk/layout : es2015 as esm2015
Compiling @angular/platform-browser/animations : es2015 as esm2015
Compiling @angular/cdk/scrolling : es2015 as esm2015
Compiling @angular/cdk/text-field : es2015 as esm2015
Compiling @angular/router : es2015 as esm2015
Compiling @angular/cdk/stepper : es2015 as esm2015
Compiling @angular/cdk/table : es2015 as esm2015
Compiling @angular/cdk/tree : es2015 as esm2015
Compiling @angular/material/core : es2015 as esm2015
Compiling @angular/fire/auth : es2015 as esm2015
Compiling @angular/cdk/overlay : es2015 as esm2015
Compiling @angular/platform-browser-dynamic : es2015 as esm2015
Compiling @angular/platform-browser/testing : es2015 as esm2015
Compiling @angular/material/expansion : es2015 as esm2015
Compiling @angular/common/testing : es2015 as esm2015
Compiling @angular/cdk/drag-drop : es2015 as esm2015
Compiling @angular/material/divider : es2015 as esm2015
Compiling @angular/material/button : es2015 as esm2015
Compiling @angular/material/checkbox : es2015 as esm2015
Compiling @angular/material/form-field : es2015 as esm2015
Compiling @angular/material/icon : es2015 as esm2015
Compiling @angular/material/dialog : es2015 as esm2015
Compiling @angular/material/progress-spinner : es2015 as esm2015
Compiling @angular/material/slide-toggle : es2015 as esm2015
Compiling @angular/material/tooltip : es2015 as esm2015
Compiling @angular/material/menu : es2015 as esm2015
Compiling @angular/material/input : es2015 as esm2015
Compiling @angular/material/tabs : es2015 as esm2015
Compiling @angular/material/card : es2015 as esm2015
Compiling @angular/material/select : es2015 as esm2015
Compiling @angular/material/badge : es2015 as esm2015
Compiling @angular/material/button-toggle : es2015 as esm2015
Compiling @angular/material/bottom-sheet : es2015 as esm2015
Compiling @angular/material/autocomplete : es2015 as esm2015
Compiling @angular/material/grid-list : es2015 as esm2015
Compiling @angular/material/chips : es2015 as esm2015
Compiling @angular/material/list : es2015 as esm2015
Compiling @angular/material/progress-bar : es2015 as esm2015
Compiling @angular/material/paginator : es2015 as esm2015
Compiling @angular/material/datepicker : es2015 as esm2015
Compiling @angular/material/radio : es2015 as esm2015
Compiling @angular/material/sidenav : es2015 as esm2015
Compiling @angular/material/slider : es2015 as esm2015
Compiling @angular/material/sort : es2015 as esm2015
Compiling @angular/material/snack-bar : es2015 as esm2015
Compiling @angular/material/stepper : es2015 as esm2015
Compiling @angular/material/table : es2015 as esm2015
Compiling @angular/material/toolbar : es2015 as esm2015
Compiling @angular/cdk-experimental/clipboard : es2015 as esm2015
Compiling @angular/material/tree : es2015 as esm2015
Compiling @angular/cdk-experimental/popover-edit : es2015 as esm2015
Compiling @angular/cdk-experimental/dialog : es2015 as esm2015
Compiling @angular/cdk-experimental/scrolling : es2015 as esm2015
Compiling @angular/common/http/testing : es2015 as esm2015
Compiling @angular/fire/database : es2015 as esm2015
Compiling @angular/fire/auth-guard : es2015 as esm2015
Compiling @angular/fire/database-deprecated : es2015 as esm2015
Compiling @angular/fire/functions : es2015 as esm2015
Compiling @angular/fire/firestore : es2015 as esm2015
Compiling @angular/fire/messaging : es2015 as esm2015
Compiling @angular/fire/performance : es2015 as esm2015
Compiling @angular/fire/storage : es2015 as esm2015
Compiling @angular/material-experimental : es2015 as esm2015
Compiling @angular/material-experimental/mdc-autocomplete : es2015 as esm2015
Compiling @angular/material-experimental/mdc-card : es2015 as esm2015
Compiling @angular/material-experimental/mdc-helpers : es2015 as esm2015
Compiling @angular/material-experimental/mdc-button : es2015 as esm2015
Compiling @angular/material-experimental/mdc-radio : es2015 as esm2015
Compiling @angular/material-experimental/mdc-checkbox : es2015 as esm2015
Compiling @angular/material-experimental/mdc-menu : es2015 as esm2015
Error: Error on worker angular/angular-cli#5: Error: Unable to write a reference to MatMenu$1 in C:/Lander/WS/acro-companion/node_modules/@angular/material-experimental/esm201
5/mdc-menu.js from C:/Lander/WS/acro-companion/node_modules/@angular/material-experimental/mdc-menu/typings/module.d.ts
    at ReferenceEmitter.emit (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\src\ngtsc\imports\src\emitter.js:54:19)
    at Object.toR3Reference (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\src\ngtsc\annotations\src\util.js:165:31)
    at NgModuleDecoratorHandler._toR3Reference (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\src\ngtsc\annotations\src\ng_module.js:348:31
)
    at C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\src\ngtsc\annotations\src\ng_module.js:173:83
    at Array.map (<anonymous>)
    at NgModuleDecoratorHandler.analyze (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\src\ngtsc\annotations\src\ng_module.js:173:48)
    at Object.analyzeDecorators (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\analysis\util.js:75:38)
    at DecorationAnalyzer.analyzeClass (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\analysis\decoration_analyzer.js:138:40)
    at C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\analysis\decoration_analyzer.js:131:55
    at Array.map (<anonymous>)
    at ClusterMaster.onWorkerMessage (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\execution\cluster\master.js:158:27)
    at C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\execution\cluster\master.js:46:95
    at ClusterMaster.<anonymous> (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\execution\cluster\master.js:238:57)
    at step (C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:136:27)
    at Object.next (C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:117:57)
    at C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:110:75
    at new Promise (<anonymous>)
    at Object.__awaiter (C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:106:16)
    at EventEmitter.<anonymous> (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\execution\cluster\master.js:232:32)
    at EventEmitter.emit (events.js:209:13)
Compiling @angular/material-experimental/mdc-chips : es2015 as esm2015
Compiling @angular/material-experimental/mdc-slide-toggle : es2015 as esm2015
Compiling @angular/material : es2015 as esm2015
Compiling @angular/material-experimental/mdc-slider : es2015 as esm2015
Compiling @angular/material-experimental/popover-edit : es2015 as esm2015
Compiling @angular/material-experimental/mdc-tabs : es2015 as esm2015
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! acro-companion@1.0.0 postinstall: `ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the acro-companion@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Lander\AppData\Roaming\npm-cache\_logs\2019-11-10T12_44_15_984Z-debug.log
× Package install failed, see above.
The update failed. See above.
× Migration failed. See above for further details.

**Comment by LanderBeeuwsaert** [comment]
So after rebooting, removing node_modules again, disabling virus scans & indexing, I ran it for a third and fourth time. It's still the same, still crashing on the same mdc.

Then I saw that angular/material-experimental stayed 8.2.3 in package.json so I decided to set it to 9.0.0-rc.0 myself before running the upgrade
=> hoorah, this solved the issue with the mdc_menu.

Only to actually have the same issue a couple of lbraries later in the upgrade process on dragula (2.1.1 so latest version):

C:\Lander\WS\acro-companion>ng update @angular/cli @angular/core --next
The installed Angular CLI version is older than the latest published version.
Installing a temporary version to perform the update.
Installing packages for tooling via npm.
C:\Users\Lander\AppData\Local\Temp\.ng-temp-packages-3nuhyo\ng -> C:\Users\Lander\AppData\Local\Temp\.ng-temp-packages-3nuhyo\node_modules\@angular\cli\bin\
ng

> @angular/cli@9.0.0-rc.1 postinstall C:\Users\Lander\AppData\Local\Temp\.ng-temp-packages-3nuhyo\node_modules\@angular\cli
> node ./bin/postinstall/script.js

+ @angular/cli@9.0.0-rc.1
added 254 packages from 206 contributors in 13.175s
Installed packages for tooling via npm.
Using package manager: 'npm'
Collecting installed dependencies...
Found 88 dependencies.
Fetching dependency metadata from registry...
                  Package "@angular/core" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/common" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/compiler" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/forms" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/platform-browser" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/platform-browser-dynamic" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/router" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/compiler-cli" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/animations" has a missing peer dependency of "tslib" @ "^1.10.0".
                  Package "@angular/service-worker" has a missing peer dependency of "tslib" @ "^1.10.0".
    Updating package.json with dependency @angular/cli @ "9.0.0-rc.1" (was "8.3.15")...
    Updating package.json with dependency @angular/core @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/language-service @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency typescript @ "3.6.4" (was "3.5.3")...
    Updating package.json with dependency @angular/common @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/compiler @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular-devkit/build-angular @ "0.900.0-rc.1" (was "0.803.18")...
    Updating package.json with dependency @angular/forms @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/platform-browser @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/platform-browser-dynamic @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/router @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/compiler-cli @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/animations @ "9.0.0-rc.1" (was "8.2.13")...
    Updating package.json with dependency @angular/service-worker @ "9.0.0-rc.1" (was "8.2.13")...
UPDATE package.json (7051 bytes)
√ Packages installed successfully.
** Executing migrations of package '@angular/cli' **

> Update an Angular CLI project to version 9.
UPDATE angular.json (20187 bytes)
UPDATE tsconfig.json (576 bytes)
UPDATE src/tsconfig.app.json (261 bytes)
UPDATE package.json (7048 bytes)
√ Packages installed successfully.
√ Migration succeeded.

> Update lazy loading syntax to use dynamic imports.
√ Migration succeeded.

** Executing migrations of package '@angular/core' **

> Static flag migration.
  Removes the `static` flag from dynamic queries.
  As of Angular 9, the "static" flag defaults to false and is no longer required for your view and content queries.
  Read more about this here: https://v9.angular.io/guide/migration-dynamic-flag
UPDATE src/app/create/modals/components/one-individual.component.ts (26328 bytes)
UPDATE src/app/create/modals/nat-individuals-selector.component.ts (6859 bytes)
UPDATE src/app/generic-components/sheet-purchase/sheet-purchase.component.ts (22080 bytes)
UPDATE src/app/organisation-admin/coach-sheet-list/coach-sheet-list.component.ts (27586 bytes)
UPDATE src/app/generic-components/sheet-search/sheet-search.component.ts (31524 bytes)
UPDATE src/app/judging/dj-judge-component/dj-judge.component.ts (47249 bytes)
UPDATE src/app/organisation-admin/sheet-overview/sheet-overview.component.ts (10625 bytes)
UPDATE src/app/organisation-admin/organisation-admin/organisation-admin.component.ts (27972 bytes)
UPDATE src/app/create/sheet-view.component.ts (20498 bytes)
UPDATE src/app/create/modals/group-mount-selector.component.ts (36269 bytes)
UPDATE src/app/create/modals/save-sheet-modal.component.ts (8131 bytes)
UPDATE src/app/create/modals/create-new-sheet.component.ts (32331 bytes)
UPDATE src/app/judging/ej-judge-component/ej-judge.component.ts (34719 bytes)
UPDATE src/app/judging/cjp-judge-component/cjp-judge.component.ts (33649 bytes)
UPDATE src/app/create/modals/pair-mount-selector.component.ts (32951 bytes)
UPDATE src/app/create/modals/dynamic-selector.component.ts (42016 bytes)
UPDATE src/app/create/modals/group-static-selector.component.ts (61607 bytes)
UPDATE src/app/create/modals/dynamic-double-landing.component.ts (50012 bytes)
UPDATE src/app/app.component.ts (14647 bytes)
UPDATE src/app/create/print/print-sheet-view.component.ts (9569 bytes)
UPDATE src/app/load/load.component.ts (28157 bytes)
UPDATE src/app/personal-invoice-list/personal-invoice-list.component.ts (8200 bytes)
UPDATE src/app/login/sign-in/sign-in.ts (23099 bytes)
UPDATE src/app/login/register/register.ts (21131 bytes)
UPDATE src/app/login/forgot-password/forgot-password.ts (8301 bytes)
UPDATE src/app/developer-admin/print-preview/print-preview.component.ts (37490 bytes)
UPDATE src/app/application-admin/user-admin/user-admin.component.ts (18390 bytes)
UPDATE src/app/application-admin/error-admin/error-admin.component.ts (19591 bytes)
UPDATE src/app/application-admin/create-free-form-invoice/create-free-form-invoice.component.ts (13559 bytes)
UPDATE src/app/application-admin/invoice-admin/invoice-admin.component.ts (20966 bytes)
UPDATE src/app/judging/kiss-judge-component/kiss-judge.component.ts (9209 bytes)
UPDATE src/app/judging/aj-judge-component/aj-judge.component.ts (27247 bytes)
UPDATE src/app/organisation-admin/session-panel-list/session-panel-list.component.ts (11102 bytes)
UPDATE src/app/organisation-admin/competition-administration/competition-administration.component.ts (19933 bytes)
UPDATE src/app/organisation-admin/panel-form/panel-form.component.ts (13625 bytes)
UPDATE src/app/organisation-admin/federation-administration/federation-administration.component.ts (19159 bytes)
UPDATE src/app/competition-calendar/competition-calendar/competition-calendar.component.ts (26194 bytes)
UPDATE src/app/artistic/artistic-create-sheet/artistic-create-sheet.component.ts (7240 bytes)
UPDATE src/app/artistic/artistic-judging/artistic-judging.component.ts (7375 bytes)
UPDATE src/app/tumbling/tumbling-create-sheet/tumbling-create-sheet.component.ts (7505 bytes)
UPDATE src/app/tumbling/tumbling-judging/tumbling-judging.component.ts (7112 bytes)
UPDATE src/app/titlebar/titlebar.component.ts (28302 bytes)
√ Migration succeeded.

> Missing @Injectable migration.
  In Angular 9, enforcement of @Injectable decorators for DI is a bit stricter.
  Read more about this here: https://v9.angular.io/guide/migration-injectable
√ Migration succeeded.

> ModuleWithProviders migration.
  In Angular 9, the ModuleWithProviders type without a generic has been deprecated.
  This migration adds the generic where it is missing.
  Read more about this here: https://v9.angular.io/guide/migration-module-with-providers
√ Migration succeeded.

> NGCC postinstall migration.
  Adds an ngcc invocation to npm/yarn's postinstall script.
  Read more about this here: https://v9.angular.io/guide/migration-ngcc
UPDATE package.json (7154 bytes)
| Installing packages...
> acro-companion@1.0.0 postinstall C:\Lander\WS\acro-companion
> ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points

Compiling @angular/cdk/keycodes : es2015 as esm2015
Compiling @angular/service-worker/config : es2015 as esm2015
Compiling @angular/animations : es2015 as esm2015
Compiling @angular/compiler/testing : es2015 as esm2015
Compiling ng2-ace-editor : module as esm5
Compiling @angular/core : es2015 as esm2015
Compiling angular-tree-component : module as esm5
Compiling @angular/common : es2015 as esm2015
Compiling ngx-page-scroll-core : es2015 as esm2015
Compiling @angular/cdk/collections : es2015 as esm2015
Compiling @angular/core/testing : es2015 as esm2015
Compiling @angular/material-experimental/mdc-helpers : es2015 as esm2015
Compiling @angular/animations/browser : es2015 as esm2015
Compiling @angular/cdk/portal : es2015 as esm2015
Compiling @angular/cdk/platform : es2015 as esm2015
Compiling @angular/cdk/observers : es2015 as esm2015
Compiling @angular/cdk/bidi : es2015 as esm2015
Compiling @angular/cdk/accordion : es2015 as esm2015
Compiling @angular/platform-browser : es2015 as esm2015
Compiling @angular/common/http : es2015 as esm2015
Compiling @angular/fire : es2015 as esm2015
Compiling @angular/forms : es2015 as esm2015
Compiling @angular/cdk/scrolling : es2015 as esm2015
Compiling @angular/platform-browser/animations : es2015 as esm2015
Compiling @angular/cdk/a11y : es2015 as esm2015
Compiling @angular/router : es2015 as esm2015
Compiling @angular/cdk/layout : es2015 as esm2015
Compiling @angular/cdk/text-field : es2015 as esm2015
Compiling @angular/cdk/overlay : es2015 as esm2015
Compiling @angular/fire/auth : es2015 as esm2015
Compiling @angular/cdk/stepper : es2015 as esm2015
Compiling @angular/material/core : es2015 as esm2015
Compiling @angular/cdk/table : es2015 as esm2015
Compiling @angular/cdk/tree : es2015 as esm2015
Compiling @angular/platform-browser/testing : es2015 as esm2015
Compiling @angular/platform-browser-dynamic : es2015 as esm2015
Compiling @angular/common/testing : es2015 as esm2015
Compiling @angular/cdk-experimental/popover-edit : es2015 as esm2015
Compiling @angular/material/divider : es2015 as esm2015
Compiling @angular/material/button : es2015 as esm2015
Compiling @angular/material/checkbox : es2015 as esm2015
Compiling @angular/material/form-field : es2015 as esm2015
Compiling @angular/material/icon : es2015 as esm2015
Compiling @angular/material/dialog : es2015 as esm2015
Compiling @angular/material/tooltip : es2015 as esm2015
Compiling @angular/material/slide-toggle : es2015 as esm2015
Compiling @angular/material/progress-bar : es2015 as esm2015
Compiling @angular/animations/browser/testing : es2015 as esm2015
Compiling @angular/material/input : es2015 as esm2015
Compiling @angular/material/menu : es2015 as esm2015
Compiling @angular/cdk/clipboard : es2015 as esm2015
Compiling @angular/material/select : es2015 as esm2015
Compiling @angular/material/progress-spinner : es2015 as esm2015
Compiling @angular/cdk/drag-drop : es2015 as esm2015
Compiling @angular/material/tabs : es2015 as esm2015
Compiling @angular/cdk-experimental/scrolling : es2015 as esm2015
Compiling @angular/common/http/testing : es2015 as esm2015
Compiling @angular/cdk-experimental/dialog : es2015 as esm2015
Compiling @angular/fire/database : es2015 as esm2015
Compiling @angular/fire/auth-guard : es2015 as esm2015
Compiling @angular/fire/firestore : es2015 as esm2015
Compiling @angular/fire/database-deprecated : es2015 as esm2015
Compiling @angular/fire/functions : es2015 as esm2015
Compiling @angular/fire/messaging : es2015 as esm2015
Compiling @angular/fire/performance : es2015 as esm2015
Compiling @angular/fire/storage : es2015 as esm2015
Compiling @angular/material/card : es2015 as esm2015
Compiling @angular/material/badge : es2015 as esm2015
Compiling @angular/material/button-toggle : es2015 as esm2015
Compiling @angular/material/autocomplete : es2015 as esm2015
Compiling @angular/material/bottom-sheet : es2015 as esm2015
Compiling @angular/material/chips : es2015 as esm2015
Compiling @angular/material/expansion : es2015 as esm2015
Compiling @angular/material/grid-list : es2015 as esm2015
Compiling @angular/material/datepicker : es2015 as esm2015
Compiling @angular/material/list : es2015 as esm2015
Compiling @angular/material/radio : es2015 as esm2015
Compiling @angular/material/paginator : es2015 as esm2015
Compiling @angular/material/sidenav : es2015 as esm2015
Compiling @angular/material/slider : es2015 as esm2015
Compiling @angular/material/snack-bar : es2015 as esm2015
Compiling @angular/material/sort : es2015 as esm2015
Compiling @angular/material/toolbar : es2015 as esm2015
Compiling @angular/material-experimental/mdc-autocomplete : es2015 as esm2015
Compiling @angular/material/stepper : es2015 as esm2015
Compiling @angular/material/table : es2015 as esm2015
Compiling @angular/material-experimental/mdc-card : es2015 as esm2015
Compiling @angular/material/tree : es2015 as esm2015
Compiling @angular/material-experimental/mdc-button : es2015 as esm2015
Compiling @angular/material-experimental/mdc-checkbox : es2015 as esm2015
Compiling @angular/material-experimental/mdc-select : es2015 as esm2015
Compiling @angular/material-experimental/mdc-chips : es2015 as esm2015
Compiling @angular/material-experimental/mdc-radio : es2015 as esm2015
Compiling @angular/material-experimental/mdc-menu : es2015 as esm2015
Compiling @angular/material-experimental/mdc-sidenav : es2015 as esm2015
Compiling @angular/material-experimental/mdc-progress-bar : es2015 as esm2015
Compiling @angular/material-experimental/mdc-slide-toggle : es2015 as esm2015
Compiling @angular/router/testing : es2015 as esm2015
Compiling @angular/material-moment-adapter : es2015 as esm2015
Compiling @angular/material-experimental/mdc-slider : es2015 as esm2015
Compiling @angular/material-experimental/popover-edit : es2015 as esm2015
Compiling @angular/platform-browser-dynamic/testing : es2015 as esm2015
Compiling @angular/material-experimental/mdc-tabs : es2015 as esm2015
Compiling @angular/service-worker : es2015 as esm2015
Compiling angular2-signaturepad : main as commonjs
Compiling angular-split : es2015 as esm2015
Compiling mobx-angular : module as esm5
Compiling ng2-dragula : es2015 as esm2015
Compiling ng2-dragula : es2015 as esm2015
Error: Error on worker #5: Error: Tried to overwrite C:/Lander/WS/acro-companion/node_modules/ng2-dragula/dist/components/dragula.service.d.ts.__ivy_ngcc_ba
k with an ngcc back up file, which is disallowed.
    at NewEntryPointFileWriter.InPlaceFileWriter.writeFileAndBackup (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\writing\in_plac
e_file_writer.js:36:23)
    at NewEntryPointFileWriter.writeFile (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\writing\new_entry_point_file_writer.js:63:
53)
    at C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\writing\new_entry_point_file_writer.js:45:69
    at Array.forEach (<anonymous>)
    at NewEntryPointFileWriter.writeBundle (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\writing\new_entry_point_file_writer.js:4
5:30)
    at ClusterWorker.compile (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\main.js:166:32)
    at Worker.<anonymous> (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\execution\cluster\worker.js:41:42)
    at Worker.emit (events.js:209:13)
    at process.<anonymous> (internal/cluster/worker.js:30:12)
    at process.emit (events.js:209:13)
    at ClusterMaster.onWorkerMessage (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\execution\cluster\master.js:158:27)
    at C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\execution\cluster\master.js:46:95
    at ClusterMaster.<anonymous> (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\execution\cluster\master.js:238:57)
    at step (C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:136:27)
    at Object.next (C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:117:57)
    at C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:110:75
    at new Promise (<anonymous>)
    at Object.__awaiter (C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:106:16)
    at EventEmitter.<anonymous> (C:\Lander\WS\acro-companion\node_modules\@angular\compiler-cli\ngcc\src\execution\cluster\master.js:232:32)
    at EventEmitter.emit (events.js:209:13)
Compiling ng2-odometer : main as commonjs
Compiling ngx-image-compress : es2015 as esm2015
Compiling ngx-page-scroll : es2015 as esm2015
Compiling ngx-contextmenu : es2015 as esm2015
Compiling ngx-markdown : es2015 as esm2015
Compiling ngx-mat-select-search : es2015 as esm2015
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! acro-companion@1.0.0 postinstall: `ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the acro-companion@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Lander\AppData\Roaming\npm-cache\_logs\2019-11-10T18_46_24_105Z-debug.log
× Package install failed, see above.
The update failed. See above.
× Migration failed. See above for further details.

**Comment by LanderBeeuwsaert** [comment]
(whoops pushed the wrong button)

**Comment by Splaktar** [comment]
OK, yeah the experimental versions don't auto update as part of `ng update`. You would need to bump those yourself.

For ng2-dragula, the error for easier reading is
```ts
Error: Error on worker #5: 
Error: Tried to overwrite C:/Lander/WS/acro-companion/node_modules/ng2-dragula/dist/components/dragula.service.d.ts.__ivy_ngcc_bak 
with an ngcc back up file, which is disallowed.
at NewEntryPointFileWriter.InPlaceFileWriter.writeFileAndBackup (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\writing\in_plac
e_file_writer.js:36:23)
at NewEntryPointFileWriter.writeFile (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\writing\new_entry_point_file_writer.js:63:53)
at C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\writing\new_entry_point_file_writer.js:45:69
at Array.forEach ()
```

**Comment by Splaktar** [comment]
Note that ng2-dragula is tested and validated with ngcc in this repo:
https://github.com/angular/ngcc-validation

**Comment by IgorMinar** [comment]
it is odd that the `material-experimental/mdc-menu` fails, isn't it?

```
Compiling @angular/material-experimental/mdc-menu : es2015 as esm2015
Error: Error on worker angular/angular-cli#5: Error: Unable to write a reference to MatMenu$1 in C:/Lander/WS/acro-companion/node_modules/@angular/material-experimental/esm201
5/mdc-menu.js from C:/Lander/WS/acro-companion/node_modules/@angular/material-experimental/mdc-menu/typings/module.d.ts
at ReferenceEmitter.emit (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\src\ngtsc\imports\src\emitter.js:54:19)
at Object.toR3Reference (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\src\ngtsc\annotations\src\util.js:165:31)
at NgModuleDecoratorHandler._toR3Reference (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\src\ngtsc\annotations\src\ng_module.js:348:31
)
at C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\src\ngtsc\annotations\src\ng_module.js:173:83
at Array.map ()
at NgModuleDecoratorHandler.analyze (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\src\ngtsc\annotations\src\ng_module.js:173:48)
at Object.analyzeDecorators (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\analysis\util.js:75:38)
at DecorationAnalyzer.analyzeClass (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\analysis\decoration_analyzer.js:138:40)
at C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\analysis\decoration_analyzer.js:131:55
at Array.map ()
at ClusterMaster.onWorkerMessage (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\execution\cluster\master.js:158:27)
at C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\execution\cluster\master.js:46:95
at ClusterMaster. (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\execution\cluster\master.js:238:57)
at step (C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:136:27)
at Object.next (C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:117:57)
at C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:110:75
at new Promise ()
at Object.__awaiter (C:\Lander\WS\acro-companion\node_modules\tslib\tslib.js:106:16)
at EventEmitter. (C:\Lander\WS\acro-companion\node_modules@angular\compiler-cli\ngcc\src\execution\cluster\master.js:232:32)
at EventEmitter.emit (events.js:209:13)
```

**Comment by LanderBeeuwsaert** [comment]
Note that ng2-dragula is tested and validated with ngcc in this repo:
=> the thing is that I already during beta tested out IVY few times. This was without ng update. And then the compilation of both mdc-menu and dragula went ok. So it's not purely ngcc I suppose. It's maybe some interaction with ng update?

Last try yesterday, I closed my IDE (webstorm) and ran it from command prompt with admin privileges. But even then it failed.

**Comment by gkalpak** [comment]
There is known issue with `ng2-dragula`, but it should not be hit with the default/recommended way of running `ngcc`. Maybe `ng update` somehow changes that.

Can you try adding an `ngcc.config.js` file at the root of your project wiith the following content (and then try to update again):

```js
module.exports = {
  packages: {
    'ng2-dragula': {
      entryPoints: {
        './dist': {ignore: true},
      },
    },
  },
};
```

**Comment by petebacondarwin** [comment]
@gkalpak a candiate for default ngcc config?

**Comment by Splaktar** [comment]
> it is odd that the `material-experimental/mdc-menu` fails, isn't it?

That appears to have been due to mismatched versions.

From https://github.com/angular/angular/issues/33718#issuecomment-552222182

> Then I saw that angular/material-experimental stayed 8.2.3 in package.json so I decided to set it to 9.0.0-rc.0 myself before running the upgrade
> => hoorah, this solved the issue with the mdc_menu.

**Comment by LanderBeeuwsaert** [comment]
small update:
removing ng-dragula fixed the issue for dragula.

I had some extra problems with material but that was due to old imports (straight from material and not from the component itself)

the mdc-menu issue was related maybe (not sure) to a wrong import in one of the components: import { MatSlideToggleChange } from '@angular/material-experimental/typings/esm5/mdc-slide-toggle';
but it's a different component, so yeah, I don't know.

After all that was said and done, I managed to boot IVY, but now I was getting:
https://github.com/angular/angular/issues/26436
So something with the async pipes that are not found.

Anyway, gonna let it rest for now. Feel free to close.

**Comment by gkalpak** [comment]
@LanderBeeuwsaert, can you confirm that the config in https://github.com/angular/angular/issues/33718#issuecomment-552518850 fixes the issue for you?

**Comment by LanderBeeuwsaert** [comment]
@gkalpak sorry, I just deleted ng-dragula in the branch instead of doing the config. Since then I ran into another issue after the upgrade was done and deleted the branch. So I have no idea.
